content/en/docs/Getting Started/tasks.md

```diff
@@ -0,0 +1,168 @@
+🔎  Verifying Kubernetes components...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+</pre>
+
+You can check that the cluster was successfully created with `kubectl`:
+
+```bash
+kubectl cluster-info
+```
+
+The output confirms that Kubernetes is running:
+
+<pre>
+Kubernetes control plane is running at https://127.0.0.1:39509
+CoreDNS is running at
+https://127.0.0.1:39509/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+</pre>
+
+## Install Tekton Pipelines
+
+1. To install the latest version of Tekton Pipelines, use `kubectl`:
+
+   ```bash
+   kubectl apply --filename \
+   https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+1. Monitor the installation:
+
+   ```bash
+   kubectl get pods --namespace tekton-pipelines --watch
+   ```
+
+When all components show `Running` under the `STATUS` column the installation
+is complete.
+
+Hit *Ctrl + C* to stop monitoring.
+
+## Create and run a basic task
+
+A **task**, represented in the API as an object of kind `Task`, defines a
+series of **steps** that run sequentially to perform some build work. Every
+task runs as a pod on your Kubernetes cluster, with each step running in its
+own container.
+
+1. To create a task, open your favorite editor and create a file named
+   `hello-world.yaml` with the following content:
+
+   ```yaml
+   apiVersion: tekton.dev/v1beta1
+   kind: Task
+   metadata:
+     name: hello
+   spec:
+     steps:
+       - name: echo
+         image: alpine
+         command:
+           - echo
+         args:
+           - "Hello World"
+   ```
+
+1. Apply the changes your cluster:
+
+   ```bash
+   kubectl apply --filename hello-world.yaml
+   ```
+
+     The output confirms that the task was completed successfully.
+
+     ```bash
+     task.tekton.dev/hello created
+     ```
+
+1. To run this task, you must instantiate it using `TaskRun`. Create another
+   file named `hello-world-run.yaml` with the following content:
+
+   ```yaml
+   apiVersion: tekton.dev/v1beta1
+   kind: TaskRun
+   metadata:
+     name: hello-task-run
+   spec:
+     taskRef:
+       name: hello
+   ```
+
+1. Apply the changes to your cluster to launch the task:
+
+   ```bash
+   kubectl apply --filename hello-world-run.yaml
+   ``` 
+
+1. Verify that everything worked correctly:
+
+   ```bash
+   kubectl get taskrun hello-task-run
+   ```
+
+     The output of this command shows the status of the task
+
+     <pre>
+     NAME                               SUCCEEDED    REASON       STARTTIME   COMPLETIONTIME
+	 hello-task-run          True         Succeeded    22h         22h
+     </pre>
+
+     The value `True` under `SUCCEEDED` confirms that TaskRun completed with no errors.
```

Comment:
nit: might be nice to also look at the logs at this point, something like `k logs --selector=tekton.dev/taskRun=hello-task-run` but np if that's just complicating things!

Reply:
Updated